phpstan.neon.dist

```diff
@@ -1,8 +1,9 @@
 parameters:
-    level: 0
+    level: 6
     paths:
         - src/
     ignoreErrors:
         - '#Instantiated class APCIterator not found\.#'
```

Comment:
todo: wenn die extensions dann in der action verfügbar sind sollten diese ignores hier raus können